100DaysSmiling/ShareViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ShareViewController.swift
+//  100DaysSmiling
+//
+//  Created by Dunya Kirkali on 12/07/2019.
+//  Copyright © 2019 Felipe Florencio Garcia. All rights reserved.
+//
+
+import UIKit
+import SwiftyUserDefaults
+
+// TODO: (dunyakirkali) Move
+extension DefaultsKeys {
+    static let startDate = DefaultsKey<Date>("start_date", defaultValue: Date())
+}
+
+class ShareViewController: UIViewController {
+    @IBAction func didTapShare(_ sender: Any) {
+        let day = numberOfDays()
```

Comment:
Perhaps have this in a function and not inside the ibaction itself

Reply:
Happy now?